Orabot.Core/SlashCommands.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Discord;
+using System.Collections.Generic;
+
+namespace Orabot.Core
+{
+	internal static class SlashCommands
+	{
+		private static ApplicationCommandOptionChoiceProperties[] documentationVersionChoices => new[]
```

Comment:
Isn't this missing an explicit option for `release`?

Reply:
I had left it out as redundant (the default is release), but I added it now.